lab-anthony/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Good work! It's always important to include a package.json if you're requiring in modules.